deploy/docker/backup/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM alpine:3.19
+
+RUN apk add --no-cache postgresql-client bash tar dcron
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)